src/reader.rs

```diff
@@ -18,7 +30,7 @@ pub struct CarReader<R> {
 
 impl<R> CarReader<R>
 where
-    R: AsyncRead + Send + Unpin,
+    R: AsyncRead + CarReaderSend + Unpin,
```

Comment:
in my tests `Send` can be entirely removed from these trait bounds, so no need for `CarReaderSend` trait

Reply:
removed